src/app/src/main/kotlin/org/vaccineimpact/api/app/security/TokenVerifyingConfigFactory.kt

```diff
@@ -29,7 +27,7 @@ class TokenVerifyingConfigFactory(
     override fun build(vararg parameters: Any?): Config
     {
         clients.forEach {
-            it.addAuthorizationGenerator({ _, profile -> extractPermissionsFromToken(profile) })
+            it.addAuthorizationGenerator(MontaguAuthorizationGenerator(repositoryFactory))
```

Comment:
Is there a missing test that this factory class is adding the expected authorization generators? (Missing anyway, before this behaviour change..)

Reply:
Overall the API test coverage isn't looking great, it's coming back at only around 30% coverage! But yeah this seems like a good point to add in a test for this class.